introduction.md

```diff
@@ -1,8 +1,9 @@
 # Firefox Data Documentation
```

Comment:
This is the document that people land on when following links to docs.tmo.
Can we add the details you added to README.md here as well to make them visible on the landing page?